code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -359,7 +359,7 @@
 
 /datum/spellbook_entry/item/armor
 	name = "Mastercrafted Armor Set"
-	desc = "An artefact suit of armor that allows you to cast spells while providing more protection against attacks and the void of space."
+	desc = "An artefact suit of armor that allows you to cast spells while providing more protection against attacks and the void of space. Additionally provides anti magic protection, though this may interfere with magic you cast on yourself (i.e Mutate or wands)."
```

Comment:
```suggestion
	desc = "An artifact suit of armor that allows you to cast spells while providing more protection against attacks and the void of space. Additionally provides anti-magic protection, though this may interfere with magic you cast on yourself (i.e Mutate or wands)."
```

Reply:
Might as well correct it since some person misspelled artifact years ago

Reply:
Technically it's correct in British English, but I guess this is ss13, we're all bloody american here ain't we.